src/util/propertyTreeHooks.ts

```diff
@@ -2,26 +2,24 @@ import { useMemo } from 'react';
 
 import { useStringProperty } from '@/hooks/properties';
 import { useAppSelector } from '@/redux/hooks';
-import { Uri } from '@/types/types';
+import { PropertyOwner } from '@/types/types';
 
 import { NavigationAimKey, NavigationAnchorKey } from './keys';
-import { hasInterestingTag, sgnUri } from './propertyTreeHelpers';
+import { sgnUri } from './propertyTreeHelpers';
 
-export function useInterestingTagOwners() {
+/**
+ * Get all the nodes marked in the profile, as a memoized list of property owners.
```

Comment:
```suggestion
 * Get all the nodes marked in the profile, as a list of property owners.
```
Don't think we need to worry about the memoization outside of this function